src/prng/mod.rs

```diff
@@ -318,11 +318,10 @@
 
 pub mod chacha;
 pub mod hc128;
-mod xorshift;
 
 pub use self::chacha::ChaChaRng;
 pub use self::hc128::Hc128Rng;
-pub use self::xorshift::XorShiftRng;
+pub use ::rand_xorshift::XorShiftRng;
```

Comment:
Ideally this should use `deprecated::XorShiftRng`. And then `SmallRng` will need to change the import path to avoid deprecation warnings.

Reply:
Also the deprecation warning itself should suggest the new path